sdk-codegen/codegen-resources/codewhispererruntime/service-2.json

```diff
@@ -1472,7 +1472,8 @@
                 "ideCategory":{"shape":"IdeCategory"},
                 "operatingSystem":{"shape":"OperatingSystem"},
                 "product":{"shape":"UserContextProductString"},
-                "clientId":{"shape":"UUID"}
+                "clientId":{"shape":"UUID"},
+                "ideVersion": {"shape":"String"}
```

Comment:
any reason why this can't just be merged service side from the user agent?

Reply:
The user agent were probably lost after many re-routing at services end, also VSC overrides the user agent.